skype/skype-ps/skype/Set-CsCceApplianceStatus.md

```diff
@@ -13,9 +13,7 @@ Provide the topic introduction here.
 ## SYNTAX
 
 ```
-Set-CsCceApplianceStatus [[-Identity] <Object>] [-Status <Object>] [-BypassDualWrite <Object>] [-Confirm]
- [-Error <Object>] [-Force] [-Instance <Object>] [-Tenant <Object>] [-WhatIf] [-AsJob] [-Version <Object>]
- [<CommonParameters>]
+Set-CsCceApplianceStatus [[-Identity] <Object>] [-Status <Object>] [-Confirm] [-Error <Object>] [-Force] [-Instance <Object>] [-Tenant <Object>] [-WhatIf] [-AsJob] [-Version <Object>] [<CommonParameters>]
```

Comment:
As with others, the parameter types are not correct.  For example, Identify is type UserIdParameter